action.yml

```diff
@@ -39,37 +39,59 @@ runs:
         fi
       shell: bash
 
-    - name: Check for /jira command in PR comments
-      if: github.event_name == 'issue_comment' && github.event.issue.pull_request != null
+    - name: Check for /jira command in PR comments and descriptions
+      if: ${{ github.event_name == 'issue_comment' && github.event.issue.pull_request != null || github.event_name == 'pull_request_target' && github.event.action == 'edited' }}
```

Comment:
Two thoughts:

1. Please add parentheses.
2. Don't we want to run this also on created pull requests? I expect people to add the key while creating a PR.

Reply:
In the reduplication of the code I've made the execution of the check run on all `pull_request_target` events, which includes `created` and `edited` (amongst others).